src/neuron/container/node.hpp

```diff
@@ -51,6 +51,28 @@ struct Diagonal {
     using type = double;
 };
 
+/**
+ * @brief
+ */
+struct FastIMemSavD {
+    static constexpr bool optional = true;
+    using type = double;
+    constexpr type default_value() const {
+        return -1e9;
```

Comment:
Could be removed if everything is passing.
With this PR the storage vectors for fast_imem-related quantities can live longer than the old ones do; the hope is that these "large" defaults might help identify cases where explicit initialisation logic is missing that used to be saved by re-allocation and re-zeroing of these vectors.

Reply:
```
++ python /gpfs/bbp.cscs.ch/ssd/gitlab_map_jobs/bbpcihpcsim/P130919/hpc/sim/blueconfigs/ci/compare_sonata_reports.py /gpfs/bbp.cscs.ch/project/proj12/jenkins/cellular/circuit-hip-v6/simulation-quick-delayconn/summation_cell_currents_v2_oneapi_2022.2.1_FastDebug.h5 output_55aac866/summation_cell_currents.h5
>>>> Different values for node id 36770 in population All:
[0(ms)] ref 0.0000000000000000 vs output -5151999918080.0000000000000000 for element_id index 0 (abs. diff 5.2e+12 rel. diff 1.0e+00)
>>>> Different values for node id 85731 in population All:
[0(ms)] ref 0.0000000000000000 vs output -4235000020992.0000000000000000 for element_id index 0 (abs. diff 4.2e+12 rel. diff 1.0e+00)
>>>> Different values for node id 445875 in population All:
[0(ms)] ref 0.0000000000000000 vs output -2478000046080.0000000000000000 for element_id index 0 (abs. diff 2.5e+12 rel. diff 1.0e+00)
```
might be a hit on this (https://bbpgitlab.epfl.ch/hpc/sim/blueconfigs/-/jobs/757244).